am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GrantAccessIntegrationTest.java

```diff
@@ -53,4 +56,17 @@ void whenCreatingDuplicateResourceAccessEntryIsOverwritten() {
 
         assertThat(databaseHelper.countExplicitPermissions(resourceId)).isEqualTo(1);
     }
+
+    @Test
+    void whenCreatingResourceForMultipleUsersShouldAppearInDatabase() {
+        Map<JsonPointer, Set<Permission>> multipleAttributePermissions =
```

Comment:
That name does not reflect real state. There are no multiple attributes in array, just one.

Why not to use `createGrantForWholeDocument` in this test? Does attribute matter for that test case? I do not think so.

Reply:
👍